docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -1378,7 +1378,8 @@ into the server truststore:
 
 *IMPORTANT* +
 You need to determine what SSL/TLS protocols you want to support. The
-example commands above uses _TLSv1.2_.
+example commands above uses _TLSv1.2_. Supported protocols include SSLv2, SSLv2Hello, SSLv3,
```

Comment:
Instead of mentioning this here and below, I think it would be better to add something like a "Configuring SSLv2Hello" section just after the  "Configuring a server SSLContext"/"Configuring a client SSLContext" sections. This section should contain a warning that indicates that the use of this protocol is discouraged.

Reply:
Similar content should also be added in authentication-client.adoc.

Reply:
I've added the sections to the documentation.